README.md

```diff
@@ -124,6 +125,7 @@ cmake-3.7.2-Linux-x86_64.tar.gz         100%[===================>]  29,26M  3,74
 -- Local CMake v3.7.2 installed to cmake/cmake-3.7.2-Linux-x86_64
 -- Next time you source this script, no download will be necessary
 ```
+##### NOTE:-If you do not have Python installed, it will give an error. To avoid this, run this command *"sudo apt-get install python"* or *"sudo apt-get install python3"* as per your requirement. ALso, after this , even cmake can give some error. You can avoid it by running this command "sudo snap install cmake". Hope this will run for all perfectly :) 
```

Comment:
As described previously, I think this paragraph should be reverted.

Reply:
@jounathaen Sure, no issues. Should I remove it completely?

Reply:
Yes, I don't think the Information belongs here.

Reply:
@jounathaen Okay!!